src/solver/conv_hip_implicit_gemm_v4r4_xdlops.cpp

```diff
@@ -209,10 +210,21 @@ static inline ConvSolution GetSolutionBase(const ConvolutionContext& ctx,
         ctx.general_compile_options;
     // clang-format on
 
+    result.construction_params.push_back(construction_parameters);
+
     if(ctx.direction.IsForward() || ctx.direction.IsBackwardData())
         result.invoker_factory = conv::MakeImplGemmDataInvokerFactory(ctx);
+    else
+    {
+        result.invoker_factory = [](const std::vector<Kernel>& kernels) {
+            return [=](const Handle& handle, const boost::any& primitve_params) {
+                const auto invoke_params = boost::any_cast<conv::WrWInvokeParams>(primitve_params);
+                const auto& tensors      = invoke_params.tensors;
+                handle.Run(kernels[0])(tensors.x, tensors.dy, tensors.dw);
```

Comment:
I guess this one needs SetTensor as well?

Reply:
Yes

Reply:
This solver does not need SetTensor. It does not have EBlocks tuning parameter, i.e., do not having splitting KDim optimization enabled in the kernel.